### DISCUSSION TITLE
Will Seq automatically generate a trace ID?

### DISCUSSION THREAD

**Discussion by code-jar** [datalust/seq-tickets#2237] [General]
I used OpenTelemerty Otel to record asp.net core trace information, The TraceId recorded in seq does not match the 'Activity.TraceId' in the application.

**Comment by liammclennan** [comment]
Hi @code-jar 

Seq does not generate trace or span ids, it stores whatever it is sent. 

You can try adding the [Console Exporter](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.Console/README.md) to get another look at the trace and span data that is being generated by the OpenTelemetry SDK (assuming you are using the OTEL SDK).

**Comment by code-jar** [comment]
I found the reason, the frontend service failed to forward the correct traceParent header to the backend.
